libs/documentation/src/lib/components/formly-checkbox/demos/basic/checkbox-basic.component.ts

```diff
@@ -17,7 +17,7 @@ export class CheckboxBasic {
       key: 'entity.userAgreement',
       type: 'checkbox',
       templateOptions: {
-        label: 'I agree.',
+        label: 'I agree By checking the box below you agree to th.',
```

Comment:
Just for demo, can you fix the typo in this label?

Reply:
Updated label